release-calendar/src/client/components/ChannelTable.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../stylesheets/channelTable.scss';
+import * as React from 'react';
+import {Channel} from '../../types';
+import {Fragment} from 'react';
```

Comment:
Isn't this already imported because you have `import * as React from 'react'`?

Reply:
I just have to write React.Fragment. I thought it was cleaner to just have Fragment, but if it's odd, I can do React.Fragment

Reply:
Between importing a module twice and using a namespace, I'd use the namespace.